modules/3d-tiles/src/lib/parsers/helpers/parse-3d-tile-subtree.ts

```diff
@@ -124,7 +98,7 @@ async function getExplicitBitstream(
 
   // External bitstream loading
   if (buffer.uri) {
-    const bufferUri = resolveBufferUri(buffer.uri, context?.url);
+    const bufferUri = `${context?.baseUrl || ''}/${buffer.uri}`;
```

Comment:
Nit: this looks like a possible source of errors... E.g.
- Do you want the path to start with a hard slash when no baseUrl?
- Maybe use `path.resolve()` from loader-utils?
- How about query parameters that need to be applied for Google 3D tiles. They are applied outside of this logic?

Reply:
* baseUrl have to be there because it is url of a *.subtree binary file. If subtree is not loaded this availability resource has no sense
* It works with Cesium auth tokens so I suppose it should be working with Google as well. The `context.fecth` is wrapped `fetchFile` (`(url) => fetchFile(url, fetchOptions)`)